clang/lib/Tooling/DependencyScanning/IncludeTreeActionController.cpp

```diff
@@ -457,20 +452,25 @@ void IncludeTreeBuilder::exitedInclude(Preprocessor &PP, FileID IncludedBy,
   SourceManager &SM = PP.getSourceManager();
   std::pair<FileID, unsigned> LocInfo = SM.getDecomposedExpansionLoc(ExitLoc);
 
-  // If the file includes already has a node for the current source location, it
-  // must be an import that turned out to be spurious.
-  auto &CurIncludes = IncludeStack.back().Includes;
```

Comment:
Nit: would be nice to keep a short comment here about what's going on; the code itself is understandable but a bit dense to read through just to see that it's about spurious imports.